modules/k8s-nmstate-deploying-nmstate-CLI.adoc

```diff
@@ -0,0 +1,101 @@
+// This is included in the following assemblies:
+//
+// networking/k8s_nmstate/k8s-nmstate-about-the-kubernetes-nmstate-operator.adoc
+
+:_content-type: PROCEDURE
+[id="installing-the-kubernetes-nmstate-operator-CLI_{context}"]
+= Installing the Kubernetes NMState Operator using the CLI
+Instead of using the web console, you can install the Kubernetes NMState Operator by using the {product-title} CLI (oc).
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+
+* Log in as a user with `cluster-admin` privileges.
```

Comment:
Would we make the other past tense? This is a common occurrence: 

"* You must be logged in as a user with `cluster-admin` privileges."